### ISSUE TITLE
About the stage 1 pretraining

### ISSUE THREAD

**Issue by qy-NJU** [eric-ai-lab/MiniGPT-5#33]
When I wanted to reproduce the experiment of stage 1, I found that the image loss (LDM, mentioned in the paper) couldn't converge. So I wonder if there is any trick I ignored, and I notice that you use snr loss as the default setting for the image loss which wasn't mentioned in the paper. Looking forward to you reply.

**Comment by KzZheng** [comment]
It's quite normal to observe that the image loss keeps fluctuating during the training process. You can refer to the concept of [Textual inversion](https://huggingface.co/docs/diffusers/main/en/using-diffusers/loading_adapters#textual-inversion), which needs to train one concept with several same concept images without fine-tuning Unet part. Here, every data sample includes new concepts, and we jointly learn them together. The primary objective of the image loss is to align the LLM's output feature with the conditional features of the diffusion model at a semantic level. An interesting experiment you might want to try is training the model with image loss only. This setup can demonstrate that even if the image loss fluctuates, the model is still capable of generating images that are semantically relevant.
